src/pages/Admin/AdminMainLayout.tsx

```diff
@@ -4,9 +4,10 @@ import styled from 'styled-components';
 import AdminNavbar from './AdminNavbar';
 
 const Container = styled.div`
-  max-width: 1200px;
-  padding: 0 60px;
+  width: 100vw;
+  height: 100vh;
   display: flex;
+  background: gray;
```

Comment:
maybe we can use the color from theme here

Reply:
Done!